s3-inf-ppo/laboratoria/lab07/c#/lab07/bin/Debug/lab07.exe.config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
Do we need binary `lab07.pdb` file in repository?

Reply:
https://stackoverflow.com/questions/2347335/how-could-i-ignore-bin-and-obj-folders-from-git-repository